Framework/DataHandling/src/LoadInstrument.cpp

```diff
@@ -99,6 +102,32 @@ void LoadInstrument::exec() {
   m_filename = getPropertyValue("Filename");
   m_instName = getPropertyValue("InstrumentName");
 
+  // Decide whether to use Nexus or IDF loading
+  // Note: for now, if only the instrument name is provided, the IDF loader will
+  // be used by default. This should be updated in the future, so that the
+  // Nexus loader can also take in an instrument name. Possibly by adding
+  // a "FileType" property to the LoadInstrument algorithm.
+  if (LoadGeometry::isIDF(m_filename, m_instName))
+    IDFInstrumentLoader();
+  else if (LoadGeometry::isNexus(m_filename))
```

Comment:
Are we going to do the "isNexus" from name as a separate PR. Did you decide?

Reply:
Yes i think I want to leave that for a different PR